sklearn/naive_bayes.py

```diff
@@ -569,8 +569,9 @@ def partial_fit(self, X, y, classes=None, sample_weight=None):
         # We convert it to np.float64 to support sample_weight consistently
         Y = Y.astype(np.float64, copy=False)
         if sample_weight is not None:
+            sample_weight = _check_sample_weight(sample_weight, X, dtype=None)
```

Comment:
```suggestion
            sample_weight = _check_sample_weight(sample_weight, X)
```
no need to explicitly set a parameter if it's its default value

Reply:
Thanks, done in f5a46f7b36e78f499cb56a084b663c4d350e9f11.